smalld/smalld.py

```diff
@@ -183,22 +183,22 @@ def headers(self):
             "User-Agent": f"DiscordBot (https://github.com/princesslana/smalld.py, {__version__})",
         }
 
-    def get(self, path):
-        return self.send_request("GET", path)
+    def get(self, path, params=None):
```

Comment:
Why have we added params?

At this point, it might be better for all these method to simply `*args, **kwargs` em and pass through to send_request.

Reply:
Some of the endpoints take query parameters, I thought this was overlooked?